lib/src/supabase_auth.dart

```diff
@@ -145,6 +149,7 @@ class SupabaseAuth with WidgetsBindingObserver {
 
   /// Dispose the instance to free up resources
   void dispose() {
+    _initialDeeplinkIsHandled = false;
```

Comment:
I guess this was added for the tests, but I don't think that's a good idea for normal apps. What about calling this only in tests?

Reply:
Makes sense. Let me do that.